repaygithub/cactus
# CACTUS-435 :: Fix Breadcrumb Styles

This came up as I was working on https://repayonline.atlassian.net/browse/CACTUS-435.

The problem is that the user agent stylesheets (for Chrome, at least) are overriding some styles on the `ul` in the `Breadcrumb` component, adding some unnecessary padding and margin.

### Testing
1. Open the Breadcrumb story
2. Inspect the elements in Chrome and make sure there is no extra margin or padding being added to the `ul` element in the DOM.
